### PR TITLE
Checkbox component

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | - |
| Decisions |  - |
| Animated GIF | bh11RZf8OG|

### CODE REVIEW

File: vue/components/ui/checkbox-platforme/checkbox-platforme.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+.checkbox-choice {
+    display: block;
+    line-height: 13px;
+    padding: 10px 6px 10px 6px;
+    user-select: none;
+}
+
+.checkbox-choice:hover,
+.checkbox-choice:hover ::v-deep label {
```
Comment (gcandal) [inline vue/components/ui/checkbox-platforme/checkbox-platforme.vue]: Is this line needed?

**Comment by gcandal** [comment]
Please merge with master.

File: vue/components/ui/checkbox-platforme/checkbox-platforme.vue (hunk 1 above)
Comment (3rdvision) [reply vue/components/ui/checkbox-platforme/checkbox-platforme.vue]: Updated `.checkbox-choice:hover ::v-deep label` -> `.checkbox-choice:hover label`
But it is still needded to override cursor default:

imagem

File: vue/components/ui/checkbox-platforme/checkbox-platforme.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            type: String,
+            default: false
+        }
+    },
+    methods: {
+        onClick(item) {
+            if (this.disabled || item.disabled) {
+                return;
+            }
+            this.values[item.value] ? this.removeItem(item) : this.addItem(item);
```
Comment (gcandal) [inline vue/components/ui/checkbox-platforme/checkbox-platforme.vue]: Move this to a `toggleItem` function

**Pull request merged**